packages/gatsby-plugin-sitemap/src/internals.js

```diff
@@ -20,29 +20,27 @@ export const defaultOptions = {
           siteUrl
         }
       }
-      allMarkdownRemark(
-        limit: 1000,
+      
+      allSitePage(
+        limit: 1000000000,
```

Comment:
By default if you don't include a limit it'll just return everything. So if that's what you want then just leave it out.